### PR TITLE
Update MEDICC2 to version 1.1.2

### PR DESCRIPTION
**Pull request by tlkaufmann**
Update MEDICC2 to version 1.1.2

Made some changes to the recipe so the autobump is not sufficient here

----

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

Note that the <code>@BiocondaBot please merge</code> command is now depreciated. Please just squash and merge instead.

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by tlkaufmann** [comment]
@BiocondaBot please add label

**Review by aliciaaevans — COMMENTED** [review]
I don't see any changes.

**Comment by tlkaufmann** [comment]
> I don't see any changes.

omg I am sorry, forgot to push the 2nd commit -.-

**Comment by aliciaaevans** [comment]
> > I don't see any changes.
> 
> omg I am sorry, forgot to push the 2nd commit -.-

Been there!

**Comment by aliciaaevans** [comment]
@BiocondaBot please fetch artifacts

**Comment by tlkaufmann** [comment]
Thanks for your help @aliciaaevans 🎉

File: recipes/medicc2/meta.yaml [hunk 1]
```diff
@@ -13,47 +13,53 @@ source:
 
 build:
   number: 0
+  skip: True  # [py < 38 or py >= 311]
```
[the remark this answers is not in the document]
Comment (tlkaufmann) [reply recipes/medicc2/meta.yaml:16]: ah that's good to know

**Pull request merged**